technical-planning-meeting.md

```diff
@@ -3,39 +3,44 @@
 
 ## Definition
 
-* A technical planning meeting is an informal discussion aimed at streamlining our development processes.
-* They can be as short or as long as necessary.
-* They should be requested by developers.
+* A technical planning meeting (TPM) is an informal discussion aimed at streamlining our development processes and agreeing a technical solution
+* They can be as short or as long as necessary
+* They should be requested by engineers
 
 ## When is one appropriate?
 
 * When starting a new project
 * When adding new features or functionality
-* When undertaking anything that is likely to be more than 5 days work
+* When the technical implementation is not clear for a feature iteration
+* When undertaking work that requires multiple teams to collaborate on a technical solution
 
 ## Expected Outcomes
 
-* High level documentation of the business problem and technical solution
-* Developers should have a clear understanding of what is expected
-* A set of Jiras for product owners to prioritise
-* A definition of done
+* High level documentation of the business problem, the technical solution and/ or next steps and the business value (it is clear the complexity of the task in order that we can establish the business benefits)
+* Engineers should have a clear understanding of what is expected
+* What the next steps are, this could be a set of Jiras for Pod Leads to prioritise 
+* A definition of done/ acceptance criteria
+* Pod Leads or Stakeholders should be clear what Tech Debt there is with any MVP (Minimum Viable Product)
```

Comment:
Maybe a link to the [Tech Debt](https://github.com/holidayextras/culture/blob/master/tech-debt.md) documentation?